2.1-segwit-version-1.ipynb

```diff
@@ -391,7 +527,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Example 2.1.11: Shutdown the TestWrapper (and all bitcoind instances)"
+    "### Benefits of using segwit v1 MuSig over segwit v0 P2WSH\n",
+    "\n",
+    "You can see that the transaction weight of the transaction spending the v1 MuSig output is about 30% lower than the transaction spending the v0 P2WSH output. For larger n-of-n multisig, the weight savings is even larger. Since transaction fees are based on the transaction weight, these weight savings translate directly to fee savings.\n",
+    "\n",
+    "In addition, by using a MuSig aggregate key and signature, Alice does not reveal that she is using a multisignature scheme, which is good for her privacy and security."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Example 2.1.16: Shutdown the TestWrapper (and all bitcoind instances)"
    ]
   },
   {
```

Comment:
Congratulations section is still somewhat accurate but I suspect you missed updating this part. We should add the learning about space savings/privacy at least.

Reply:
You're right. This should have been updated. Done.